diff -Naur dwm-6.1.orig/config.mk dwm-6.1.new/config.mk
--- dwm-6.1.orig/config.mk	2015-11-08 14:39:37.000000000 -0800
+++ dwm-6.1.new/config.mk	2016-02-10 17:57:24.093578761 -0800
@@ -16,9 +16,10 @@
 
 # freetype
 FREETYPELIBS = -lfontconfig -lXft
-FREETYPEINC = /usr/include/freetype2
+# FREETYPEINC = /usr/include/freetype2
 # OpenBSD (uncomment)
-FREETYPEINC = ${X11INC}/freetype2
+# FREETYPEINC = ${X11INC}/freetype2
+FREETYPEINC = /usr/local/include/freetype2
 
 # includes and libs
 INCS = -I${X11INC} -I${FREETYPEINC}
